app/mailers/sign_up_mailer.rb

```diff
@@ -25,4 +25,12 @@ def admin_notification_new_staff(tenant_org)
       to: "wcrwater@gmail.com",
       subject: "New Staff Account")
   end
+
+  def create_new_org_and_admin(tenant_org)
+    @organization = params[:organization]
+    @user = params[:user]
+    mail(from: MultiTenantService.new(tenant_org).default_email,
```

Comment:
Not sure what to use as the `from` email. I suspect we'd want this to be from pet rescue rather than the org itself.